### PR TITLE
fix 1853

### LINKED ISSUE #1853: [BUG] Environment variables containing `status:` are removed
### Expected Behavior

I want to use `status:xxx` as a value of one of envirnoment variables

### Actual Behavior

It removes `status:` and the subsequent lines in the generated `app-deployment.yaml`

### Steps To Reproduce

Run `kompose convert` with v1.32.0 with attached dock-compose file

### Kompose Version

```Text
1.32.0
```

### Docker-Compose file

```YAML
services:
  app:
    image: node:18-alpine
    command: sh -c "yarn install && yarn run dev"
    ports:
      - 3000:3000
    environment:
      TOPICS: foo:1,status:2,bar:3
      OTHER_ENV: example
```

### Anything else?

Bug seems to be introduced in this PR: https://github.com/kubernetes/kompose/pull/1746

### PR DESCRIPTION
**Pull request by sosan**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:
patch add a \n in regex  re := regexp.MustCompile(`(?s)status:\n.*`)
#### Which issue(s) this PR fixes:

Fixes #1853

#### Special notes for your reviewer:
status field in DeploymentConfig cannot be null

by default, convert include follow statement in file out-X-os.yaml
status:
  availableReplicas: 0
  latestVersion: 0
  observedGeneration: 0
  replicas: 0
  unavailableReplicas: 0
  updatedReplicas: 0

### CODE REVIEW

File: script/test/fixtures/envvars-with-status/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+services:
+  app:
+    image: node:18-alpine
+    command: sh -c "yarn install && yarn run dev"
+    ports:
+      - 3000:3000
+    environment:
+      TOPICS: foo:1,status:2,bar:3
+      OTHER_ENV: example
```
Comment (TessaIO) [inline script/test/fixtures/envvars-with-status/compose.yaml:9]: Missing EOF

**Comment by cdrage** [comment]
@sosan sorry, needs a rebase again and we are good to go

**Comment by sosan** [comment]
merged

**Pull request merged**